Maquetado/index.html

```diff
@@ -11,5 +11,13 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1> 
+  <div class="tech-border">
+    <div class="tech">
+      <img src="./assets/react_icon.svg">
+      <h2>ReactJS</h2>
+      <p><b>Release: 2013</b></p>
```

Comment:
the `b` tag represents a bold text, this should be achieved from css using `font-weight: bold` instead since CSS is used for styling and HTML for structure. This will also make the DOM a bit simpler.